Neos.ContentRepository.Core/Classes/SharedModel/Node/NodeAggregateId.php

```diff
@@ -50,6 +50,23 @@ public static function fromString(string $value): self
         return new self($value);
     }
 
+    /**
+     * Calculates a deterministic nodeAggregateId.
+     *
+     * Used for determining the nodeAggregateIds of tethered nodes.
+     *
+     * e.g. in case you want to get the main content collection node aggregate id:
+     *  NodeAggregateId::fromParentNodeAggregateIdAndNodeName($parentNodeAggregateId, NodeName::fromString('main'))
+     *
+     */
+    public static function fromParentNodeAggregateIdAndNodeName(NodeAggregateId $parentNodeAggregateId, NodeName $tetheredNodeName): self
```

Comment:
move to NodeName

Reply:
I agree that this should be moved because it contains a concept that does not belong to the Node Aggregate ID itself.. It's not too bad though IMO and I'm not sure if `NodeName` is a better place.. @skurfuerst  do you have an idea where to put this (and please don't say "utility class" ;))